src/uct/api/v2/uct_v2.h

```diff
@@ -577,6 +588,40 @@ typedef struct uct_iface_is_reachable_params {
 } uct_iface_is_reachable_params_t;
 
 
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief Operation parameters passed to @ref uct_ep_is_connected.
+ *        Only addresses with addr_len > 0 should be passed.
```

Comment:
why is this a requirement?

Reply:
to define when to pass what parameters. Otherwise it would be quite confusing. We have the same comments for is_reachable_v2